### PR TITLE
fix: MD022/blanks-around-heading

### PR DESCRIPTION
**Pull request by nschonni**
Headings should be surrounded by blank lines.
Autocleanup with `markdownlint --fix  "**/*.md"`

### CODE REVIEW

File: docs/android/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -270,6 +270,7 @@ I/monodroid-gref(27679): -g- grefc 1952 gwrefc 259 handle 0x4066df10/G from take
 ```
 
 ## Object is still alive, as handle != null
+
 ## wref turned back into a gref
```
Comment (nschonni) [inline docs/android/troubleshooting/troubleshooting.md:274]: Empty double H2 seems off

File: docs/android/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -279,6 +280,7 @@ I/monodroid-gref(27679): -w- grefc 1930 gwrefc 38 handle 0xde68f95f/W from take_
 ```
 
 ## Object is dead, as handle == null
+
 ## wref is freed, no new gref created
```
Comment (nschonni) [inline docs/android/troubleshooting/troubleshooting.md:284]: Empty double H2 seems off

**Review by nschonni — COMMENTED** [review]
Not sure if the TOC.md files should be converted to the TOC.yml format. Don't know if there is any easy automation for that right now

**Comment by conceptdev** [comment]
Yeah, the TOC.md files probably shouldn't be modified. We are gradually migrating them to YAML. Not updating them will cause warnings if we enforce the rule though. Let me think about it.

**Comment by davidbritch** [comment]
Hi @nschonni 

Apologies that this PR never got fully dealt with last year. Unfortunately, the number of merge conflicts is now large enough that I'm going to have to close this PR.

**Comment by nschonni** [comment]
You can run this yourself locally by disabling the rule locally and running the markdown-cli. You might need to `npm i -g markdownlint-cli` if you don't have it installed already

**Pull request closed without merging**